docs/data_management/lts/tutorial/shared_lts_tutorial.md

```diff
@@ -0,0 +1,48 @@
+---
+toc_depth: 3
+---
+# Basic Workflow With Shared LTS and `s3cmd`
+
+This tutorial is intended for PIs, Core Directors, as well as their managers and stewards, who are responsible for managing shared LTS allocations for their Lab or Core. While the installation of `s3cmd` is the same as covered in [our individual LTS tutorial](./individual_lts_tutorial.md#install-s3cmd-within-conda-environment-on-cheaha), this tutorial focuses on configuring and using s3cmd for shared LTS allocations especially when you manage both individual and shared LTS.
+
+We will walk through how to configure access for a shared LTS, switch between your individual and shared LTS allocations, and perform key operations such as creating and managing buckets and objects. We will also show you how to set up and manage read and write access for other allocations to your shared LTS buckets and objects.
+
+## Prerequisites
+
+To follow this tutorial effectively, you should have a basic understanding of the [prerequisites covered in our individual LTS tutorial](individual_lts_tutorial.md#prerequisites). In addition, you should have a shared LTS allocation already created by our team either with you as the owner or designated as a manager or steward for your Lab's or Core's shared LTS allocation.
+
+## Configuring `s3cmd` for Shared LTS Buckets
+
+Once you have `s3cmd` installed and your environment activated as done for in [our individual LTS tutorial](./individual_lts_tutorial.md#install-s3cmd-within-conda-environment-on-cheaha), you can begin the configuration process using the following command:
+
+`s3cmd --configure -c $HOME/<lts-profile-name>`
+
+In the same way, you can configure multiple LTS profiles if you own or manage more than one LTS allocation. Just use the corresponding access and secret keys, and replace `<lts-profile-name>` with an appropriate name for each profile.
+
+To start configuration, for example for the `my-lab` profile, run the command `s3cmd --configure -c $HOME/my-lab` in your terminal. During the configuration process, you will be prompted to enter various details, such as the Access Key, Secret Key, S3 Endpoint, and DNS-style bucket template.
+
+The screenshot below shows the full terminal output of `s3cmd --configure -c $HOME/my-lab` using the `my-lab` profile as an example. When prompted, enter the Access Key and Secret Key provided by your lab. Use `https://s3.lts.rc.uab.edu` as the S3 endpoint. For the DNS-style bucket template, enter `%(bucket).s3.lts.rc.uab.edu`. Leave the remaining fields at their default values or blank. When asked to test the credentials, type `n` for `no`. Then, type `y` to save your configuration. For more details, please refer our [generic s3cmd configuration](../interfaces.md#configuring-s3cmd) guide.
+
+![s3cmd configuration in terminal showing full interactive setup for a shared LTS bucket, including credentials and endpoint settings.](../images/shared-lts-config.png)
```

Comment:
This is very close! Please also describe the medium through which the image content would be perceived. What I mean is add something like "in a terminal".

I had to look this up: apparently backticks `` ` `` are dropped during rendering of alt-text, leaving only the enclosed content.

The intent of backticks in Markdown is to ensure inline code rendering `<code>...</code>` which is irrelevant for alt-text, because alt-text is not rendered visually.

The general preference for alt-text is to avoid unnecessary symbols, because screen readers may read them out loud literally.

So `$%&*` would be read out loud as `dollar sign percent symbol ampersand asterisk`. It is compact visually, but not compact audibly.

In short, please remove the backticks in alt-text.